File: PHP/car.php
<?php
require_once 'account.php';

class Car {

public $id;
public $license;
public $driver;
public $passenger;

public function __construct($license, $driver)
{
    $this->license = $license;
    $this->driver = $driver;
}


public function printDataCar(){

echo '<p>' . $this->license . '</p>';
echo '<p>' . $this->driver->name . '</p>';
echo '<p>' . $this->driver->document . '</p>';

}

}

?>

File: Java/Car.java


class Car {
private Integer id;
private String license;
private Account driver;
private Integer passenger;


public Car(final String license, final Account driver) {

    this.license = license;
    this.driver = driver;

}




public Integer getPassenger() {

    return passenger;
}

public void setPassenger(final Integer passenger) {

    if (passenger == 4) {

        this.passenger = 4;
    } else {

        System.out.println("Número de pasajeros incorrecto, favor asignar 4 pasajeros");
    }
}

public Integer getId() {
    return id;
}

public void setId(final Integer id) {
    this.id = id;
}

public String getLicense() {
    return license;
}

public void setLicense(final String license) {
    this.license = license;
}

public Account getDriver() {
    return driver;
}

public void setDriver(final Account driver) {
    this.driver = driver;
}

void printDataCar(){

    if(passenger != null){
System.out.println("License: " + license + " Driver Name: " + driver.name + " Passengers: " + passenger);

    }
}

}